tests/pipeline/test_engine.py

```diff
@@ -115,7 +154,8 @@ def setUp(self):
 
     def test_bad_dates(self):
         loader = self.loader
-        engine = SimplePipelineEngine(loader, self.dates, self.asset_finder)
+        engine = SimplePipelineEngine(lambda column: loader,
```

Comment:
Style nit: can we put the args here on separate lines?  (I assume they no longer fit in 80 characters on one line.)